Source/Frontend/UI/Extensions/ComponentForm.cs

```diff
@@ -0,0 +1,179 @@
+namespace RTCV.UI
```

Comment:
Could you move this into UI/Modular instead? Not sure why this was separate from that code as this is part of the modular UI system